src/main/java/edu/utdallas/davisbase/representation/WhereExpression.java

```diff
@@ -0,0 +1,30 @@
+package edu.utdallas.davisbase.representation;
+
+import net.sf.jsqlparser.expression.Expression;
+
+public class WhereExpression {
+  String expression;
+  private boolean not;
+  private String column;
+  private String operator;
```

Comment:
```suggestion
  private final String operator;
```

Also, let's make this a structured field rather than a raw string. Perhaps an enum of the valid comparison operator types that DavisBase accepts?